x-pack/plugins/apm/public/components/app/correlations/error_correlations.tsx

```diff
@@ -42,11 +42,7 @@ type CorrelationsApiResponse = NonNullable<
   APIReturnType<'GET /api/apm/correlations/errors/failed_transactions'>
 >;
 
-interface Props {
-  onClose: () => void;
-}
-
-export function ErrorCorrelations({ onClose }: Props) {
+export function ErrorCorrelations() {
```

Comment:
Can we update the terminology for this? Also the file and elsewhere

```suggestion
export function FailedTransactionCorrelations() {
```

Reply:
We are working on failed transactions in parallel in this PR (https://github.com/elastic/kibana/pull/108441) that's why I tried to touch these files as little as possible in this PR but we should definitely rename later.

Reply:
Ah, okay. Makes sense 👍